packages/ui/src/components/Badge/Badge.vue

```diff
@@ -0,0 +1,114 @@
+<script lang="ts">
+export default defineComponent({
+  name: 'SBadge',
+})
+</script>
+
+<script setup lang="ts">
+import { SSpinner } from '../Spinner'
+import { BadgeType } from './types'
+
+const props = withDefaults(
+  defineProps<{
+    type?: BadgeType
+    colorBackground?: boolean
+    withBorder?: boolean
+    onlyMarker?: boolean
+  }>(),
+  {
+    type: 'active',
+    colorBackground: false,
+    withBorder: false,
+    onlyMarker: false,
+  },
+)
+
+const showSpinner = computed(() => {
+  return props.type === 'pending'
+})
+</script>
+
+<template>
+  <div
+    tabindex="0"
+    :class="[
+      's-badge',
+      { 's-badge--border': withBorder },
+      colorBackground ? `s-badge--color-${type} text-white` : ' text-brand-pms-black',

```

Comment:
We don't have `text-branch-pms-black` utility - now our Windi config contain only default utilities.